retrieverdash/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Django==2.0.2
 pytz==2017.3
+git@github.com:weecology/retriever.git
```

Comment:
Is this working? I don't think that should work. I think `git+https://www.github.com/weecology/retriever@master` would be better.

Reply:
correct